pavlo976/BuildMinder-App
# Feature/translation

### What's changed?
Languages functions was created to get data from languages table from supabase database
### How to test it?
Import function to some component and just call it. Check if data are returned in the console on the browser.
### What's missing?
Everything is up to date.
